### DISCUSSION TITLE
Redirection after Payment Completion

### DISCUSSION THREAD

**Discussion by naveed-strativ** [pretix/pretix#4603] [Q&A] [answered]
Hello,

I’m looking for a way to redirect users to a custom URL after an order or payment is complete, ideally for manual payments. I’ve enabled “redirection from order page” in the API plugin section and checked the “order” and “invoice” API responses but couldn’t find redirect URLs there.

Is there an existing plugin or feature that supports this?

**Comment by raphaelm** [comment, marked as answer]
The "redirection from order page" only works when the entire order is created through the API, as e.g. outlined here: https://docs.pretix.eu/en/latest/api/guides/custom_checkout.html

To redirect somewhere for payment, a proper payment plugin would need to be used.

**Reply by naveed-strativ** [reply]
Thanks a lot @raphaelm. Got the redirection to work.
